### ISSUE TITLE
Problem initializing the DB on Windows 10 x64

### ISSUE THREAD

**Issue by khareamol** [yandex-qatools/postgresql-embedded#51]
My team has successfully started using this library.
However, when I work on Windows 10 x64, I found that DB is not getting initialized.

After debugging for root cause
Found that following 2 lines need to be reversed or better solution would be to --pgdata while specifying the Data directory.

InitDbProcess - Line 69, 70
```java
        ret.add(config.storage().dbDir().getAbsolutePath());
        ret.addAll(config.getAdditionalInitDbParams());
```

Since the DB Storage argument is coming before the additional parameters like -E, Windows command comes back throwing exception saying
```
initdb: too many command-line arguments (first is "-E")
```
If I remove the Additional Init DB Params that I want to set [Locale, Encoding, Collate, ...], then it works absolutely fine.

Also let me know if any workaround is possible for the same.

**Comment by smecsia** [comment]
@khareamol Feel free to send a PR with the tests covering your specific use case. I'm going to configure TravisCI with different platforms soon, so we could see if your tests would fail.

**Comment by novator007** [comment]
Hi,
I also encountered same problem. If you switch those two lines, db starts fine with additional parameters.

**Comment by smecsia** [comment]
@novator007 again, please feel free to contribute. If it wont break the compatibility with Linux and MacOS, I'll merge it with pleasure.

**Comment by melezov** [comment]
PR which resolves this is available #61

**Comment by smecsia** [comment]
@novator007 @khareamol New release version contains the fix. So I'm closing this issue. Feel free to reopen if it does not help.
